clang/include/clang/AST/RecursiveASTVisitor.h

```diff
@@ -2636,6 +2636,12 @@ DEF_TRAVERSE_STMT(ObjCBridgedCastExpr, {
 DEF_TRAVERSE_STMT(ObjCAvailabilityCheckExpr, {})
 DEF_TRAVERSE_STMT(ParenExpr, {})
 DEF_TRAVERSE_STMT(ParenListExpr, {})
+DEF_TRAVERSE_STMT(UniqueStableNameExpr, {
+  if (S->isExpr())
```

Comment:
Is this actually needed? I thought the way `DEF_TRAVERSE_STMT` works on statements is to call `children()` and traverse to all children automatically. So I kind of wonder if this means we're traversing the expression form twice.

Reply:
Ooh, I think you're right!  Thanks!